spirv_reflect.c

```diff
@@ -1214,6 +1314,9 @@ static SpvReflectResult ParseFunctions(SpvReflectPrvParser* p_parser)
       }
       p_func->callee_ptrs = (SpvReflectPrvFunction**)calloc(p_func->callee_count,
                                                             sizeof(*(p_func->callee_ptrs)));
+      if(IsNull(p_func->callee_ptrs)){
```

Comment:
Added a few allocation failure checks. But still, accessing internal members before testing for NULL before SafeFree parents still may result in segfault on creation alloc failure.